src/vs/editor/browser/dnd.ts

```diff
@@ -31,3 +36,34 @@ export function createFileDataTransferItemFromFile(file: File): IDataTransferIte
 		return new Uint8Array(await file.arrayBuffer());
 	});
 }
+
+const INTERNAL_DND_MIME_TYPES = Object.freeze([
+	CodeDataTransfers.EDITORS,
+	CodeDataTransfers.FILES,
+	DataTransfers.RESOURCES,
+]);
+
+export function addExternalEditorsDropData(dataTransfer: VSDataTransfer, dragEvent: DragEvent) {
```

Comment:
When dragging a file from the OS into a tree view I now see 2 of the same file getting added by `addExternalEditorsDropData`.